Drafts/1NNN-VLP.md

```diff
@@ -0,0 +1,86 @@
+# New proposed syntax for mixins
+
+| Field           | Value                                                           |
+|-----------------|-----------------------------------------------------------------|
+| DIP:            | (number/id -- assigned by DIP Manager)                          |
+| Review Count:   | 0 (edited by DIP Manager)                                       |
+| Author:         | Victor Porton <porton@narod.ru>                                              |
+| Implementation: | (links to implementation PR if any)                             |
+| Status:         | Will be set by the DIP manager (e.g. "Approved" or "Rejected")  |
+
+## Abstract
+
+The programmer sometimes needs to instantiate mixin templates with the same arguments repeatedly, like:
+
+```
+mixin template A(int x, float y) {
+  // ...
+}
+
+mixin template B(int x, float y) {
+  // ...
+}
+
+I propose to shorten the above code with the following syntax:
+
+template composite(int x, float y) {
+  mixin A {
+    // ...
+  }
+  mixin B {
+    // ...
+  }
+}
+
+// ...
+
```

Comment:
I believe this should be `alias ourMixins = composite!(1, 2.0)`

Is this the only way to use the shortened mixin syntax, or can you still do A!(1, 2.0) and B!(1, 2.0) directly?